README.md

```diff
@@ -463,49 +463,73 @@ You can only build HybridApp if you have been granted access to [`Mobile-Expensi
     [url "https://github.com/"]
         insteadOf = ssh://git@github.com/
     ```
-3. The first time you build the app you will need to build YAPL (OldDot javascript logic). Simply run `npm run grunt:build:shared` from the `Mobile-Expensify` submodule
```

Comment:
Just to make sure, we remove it here, because PR that automises this process was merged, right? 😅

Reply:
Oh thanks, yeah I had this as a cleanup item 👍

Reply:
I added the issue to the description